util/util_tiledb.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
+
+# ================================================================
+# This tool goes a bit beyond
+#   print(grp._dump(True))
+# by also revealing array schema.
+# ================================================================
+
+import sys, os
+import tiledb
+
+def show_single_cell_group(uri):
+    """
+    Show some summary information about an ingested TileDB Single-Cell Group.
+    """
+    print('================================================================')
+    print('X/data:')
+    with tiledb.open(uri+'/X/data') as A:
```

Comment:
`os.path.join(uri, "X", "data")`

Reply:
@nguyenv will that misbehave when this client is run on Windows?

Reply:
i'll add a comment to this effect

Reply:
OIC. Disregard this comment then.

Reply:
no it's good @nguyenv !! others will surely see the same thing & think the same thing so it's definitely worth pointing out why the explicit `/` :)

Reply:
@nguyenv there was `os.path.join` in another file, and this `+` here ... i went with `os.path.join` for consistency along with a portability note in `apis/python/README.md`